types/ember-compatibility-helpers.d.ts

```diff
@@ -0,0 +1,16 @@
+export function gte(version: string): boolean;
+export function lte(version: string): boolean;
+export const HAS_UNDERSCORE_ACTIONS: boolean;
```

Comment:
Since we don't use all these other types, do we need them?

In other words, can we just have `gte`?